person.rb

```diff
@@ -30,4 +31,9 @@ def can_use_services?
 
     false
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
-  You did a great effort by adding this method to add rental however I think this method cannot work because to add rental you need to pass into the params book and date. and also The person itself should be sent as a parameter to create the new rental achieving the 'has-many' relationship  please follow this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to learn more about the association :+1: 

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
ok changes made accordingly